### DISCUSSION TITLE
Can anyone think of a versioning solution when using Vite instead of Mix?

### DISCUSSION THREAD

**Discussion by shealan** [inertiajs/inertia#564] [Help (Vue)]
I have my Laravel/Inertia/Vue project running with Vite and Tailwind JIT. It's insanely fast!

Because Vite replaces Mix entirely I cannot use the .version() feature. This means that although template/css changes are instant. Most data or function changes in my Vue code triggers a full page reload.

Can anyone think of a solution for adding the version feature to Vite in order to activate full HMR?

With Vite becoming more popular due to the crazy speed improvements, especially paired with Tailwind JIT, I see this becoming a popular request.

**Comment by zorca** [comment]
May be: https://vitejs.dev/config/#build-manifest

**Reply by shealan** [reply]
I have that activated already... I guess there needs to be some kind of way of getting Inertia to look at that file?
